ocaml/idl/datamodel.ml

```diff
@@ -1466,7 +1466,7 @@ end
 (* These are included in vbds and vifs -- abstracted here to keep both these uses consistent *)
 let device_status_fields =
   [
-    field ~ty:Bool ~qualifier:DynamicRO "currently_attached" "is the device currently attached (erased on reboot)";
+    field ~ty:Bool ~qualifier:StaticRO ~default_value:(Some (VBool false)) "currently_attached" "is the device currently attached (erased on reboot)";
```

Comment:
please a `~lifecycle:[Changed, rel_stockholm, "explanation"]` parameter in all the changed `field`s.

Reply:
Done! :)